common/src/main/java/com/viaversion/viaversion/protocols/v1_20_3to1_20_5/rewriter/BlockItemPacketRewriter1_20_5.java

```diff
@@ -1486,8 +1487,16 @@ private void updateBlockEntityTag(final UserConnection connection, @Nullable fin
                 data.set(StructuredDataKey.BASE_COLOR, baseColorIntTag.asInt());
             }
 
-            if (tag.contains("Items")) {
-                updateItemList(connection, data, tag, "Items", StructuredDataKey.CONTAINER1_20_5, true);
+            final ListTag<CompoundTag> itemsTag = tag.getListTag("Items", CompoundTag.class);
+            if (itemsTag != null) {
+                final Item[] items = new Item[27];
```

Comment:
`Items` isn't just used for shulker boxes so you cannot presize like that, and otherwise would still need slot checks below